### ISSUE TITLE
About the reported batch size.

### ISSUE THREAD

**Issue by heyubox** [wizard-III/WizardCodeR#9]
<img width="942" height="382" alt="Image" src="https://github.com/user-attachments/assets/2d547b9e-2b5c-4a44-9186-91c19c985abc" />

I noticed that you compare the training steps with other similar-sized models. As far as I know, the other models are trained with **128** batch size, but you set the batch size **64**.  What are your actual training steps under 64 batch size? Is the listed training step normalized such that two steps with batch size 64 are equivalent to one step with batch size 128?

**Comment by wizard-III** [comment]
The training steps listed in the table are the actual steps used for our model. Since the table is meant to compare the real GPU training time, we report the final training steps directly rather than aligning them with those of other models.

**Comment by wizard-III** [comment]
> The training steps listed in the table are the actual steps used for our model. Since the table is meant to compare the real GPU training time, we report the final training steps directly rather than aligning them with those of other models.

Admittedly, I didn’t pay attention to the batch sizes of the other models at the time.
